### PR TITLE
First Version of CAMS

### PR DESCRIPTION
**Pull request by Pycomet-zz**
This PR contains all the updates done to the UI including the zoom button, stations, and data picker functionalities

### CODE REVIEW

File: src/App.js
Comment (chichengren) [inline src/App.js]: can we replace this with styling from css?

File: src/components/Header.jsx
Comment (chichengren) [inline src/components/Header.jsx]: nit: let's remove unused code

File: src/components/Header.jsx
Comment (chichengren) [inline src/components/Header.jsx]: nit: same, unused code should be removed

File: src/components/Preloader.jsx
Comment (chichengren) [inline src/components/Preloader.jsx:13]: Is this reloader from the design? I feel like this might be too over-whelming, shall we considering using a smaller & simpler spinner?

File: src/components/Stations.jsx
Comment (chichengren) [inline src/components/Stations.jsx:18]: We shall probably considering using a mapping that translation site id to it's name in the dropdown?

File: src/components/Globe.jsx
Comment (chichengren) [inline src/components/Globe.jsx]: Since context is globally available in the application. We shall define it on the top level, e.g: `App.jsx` so that it's available to other components.

**Review by chichengren — COMMENTED** [review]
it seems like there're a lot of places where the unused code is not deleted from this PR. let's make sure we clean them up before merging this one.

**Comment by Pycomet-zz** [comment]
> it seems like there're a lot of places where the unused code is not deleted from this PR. let's make sure we clean them up before merging this one.

Okay, I would attend to them one at time

File: src/components/Preloader.jsx
Comment (Pycomet-zz) [reply src/components/Preloader.jsx:13]: Fixed it

File: src/components/Globe.jsx
Comment (chichengren) [inline src/components/Globe.jsx:115]: Why do we use the length of the array instead of the original id?

File: src/components/ZoomButton.jsx
Comment (chichengren) [inline src/components/ZoomButton.jsx:20]: Do we have asset for those two buttons? I think use image / svg is preferred over character

**Review by chichengren — APPROVED** [review]
Overall things looking good to me. Please try to address other comments in this PR though I don't necessarily think they're blockers to merge. Approving now.

**Review by chichengren — APPROVED** [review]

**Pull request merged**
